src/reducers/reviewReducer.js

```diff
@@ -0,0 +1,48 @@
+import * as types from '../actions/actionTypes';
+
+const initialState = {
+    message: '',
+    error: null,
+    auth_token: localStorage.getItem('auth_token')
+};
+
+const reviewsInitialState = {
+    reviews: [],
+    message: '',
+    error: null
+};
+
+export const createReview = (state=initialState, action)  => {
+    switch (action.type) {
+        case types.CREATE_REVIEW:
```

Comment:
@tonykimathi Can you indent your switch case statements properly for code readability. Like:
```
switch (action.type) {
  case types.CREATE_REVIEW:
    return {
      ...state,
      message: action.payload,
      error: false,
    };
  case types.CREATE_REVIEW_ERROR:
    return {
      ...state,
      message: action.payload,
      error: true,
    };
  default:
    return { ...state };
}
```
For linting issues check out **[Prettier](https://prettier.io/)** for your favorite editor, It may help you.
Otherwise,  Good work.